src/install/python/python-frameworks/python-agent-fastapi.mdx

```diff
@@ -0,0 +1,19 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using FastAPI
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command, or you can use the Python agent API.
```

Comment:
```suggestion
You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command if using Python agent **version 5.22.0.151** or higher, or you can use the Python agent API. There's no specific steps you need to take to integrate with FastAPI so skip straight to the server specific instructions
```